### PR TITLE
add python environment customization guide

### PR DESCRIPTION
**Pull request by mbektas**
adding instructions to guide on how to set proper permissions and customize the python environment

### CODE REVIEW

**Review by krassowski — COMMENTED** [review]
It looks good to me! I would consider adding:
- an info on re-using existing kernels by providing a custom `kernelspec` (see https://github.com/jupyterlab/jupyterlab_app/issues/238#issuecomment-927166291 but I don't know if these steps work for others yet)
- an information on when users should customise the virtual environment (e.g. installing server extensions, or using jupyterlab-app just for a single project), and when to re-use other kernels instead (which I would recommended when there are multiple virtual environments.projects)

File: env-customization-guide.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+# Customizing the Bundled Python Environment
+
+JupyterLab App is a self-contained standalone desktop application which bundles a Python environment. The bundled Python environment comes with several popular Python libraries to make the App ready to use in scientific computing and data science workflows. These packages are `numpy`, `scipy`, `pandas`, `ipywidgets` and `matplotlib`. In order to install additional packages into JupyterLab App's Python environment, you need to follow certain steps during and after the installation as described below.
+
+## Linux Instructions
+
+On Linux, JupyterLab App is installed into `/opt/JupyterLab` and Python environment is created in `/opt/JupyterLab/resources/jlab_server`
+
+- Install the App by double clicking the installer file and wait for then installation to finish.
```
Comment (krassowski) [inline env-customization-guide.md:9]:
```suggestion
- Install the App by double clicking the installer file and wait for the installation to finish.
```

File: env-customization-guide.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+- Linux installs JupyterLab App as the root user. That's why it is necessary to change the ownership to current user to be able to customize JupyterLab App's Python environment.
```
Comment (krassowski) [inline env-customization-guide.md:10]:
```suggestion
- Linux installs JupyterLab App as the root user. That's why it is necessary to change the ownership to the current user to be able to customize JupyterLab App's Python environment.
```

**Comment by mbektas** [comment]
> It looks good to me! I would consider adding:
> 
> * an info on re-using existing kernels by providing a custom `kernelspec` (see [#238 (comment)](https://github.com/jupyterlab/jupyterlab_app/issues/238#issuecomment-927166291) but I don't know if these steps work for others yet)
> * an information on when users should customise the virtual environment (e.g. installing server extensions, or using jupyterlab-app just for a single project), and when to re-use other kernels instead (which I would recommended when there are multiple virtual environments.projects)

thanks @krassowski!, sure, lets extend the guide with follow-up PRs.

**Pull request merged**